llama-index-integrations/readers/llama-index-readers-database/llama_index/readers/database/base.py

```diff
@@ -73,16 +73,21 @@ def __init__(
                 "set of credentials."
             )
 
-    def load_data(self, query: str) -> List[Document]:
+    def load_data(self, query: str, metadata: dict=None, doc_cols: list=None, metadata_cols: list=None) -> List[Document]:
```

Comment:
```python
from typing import Dict, List

def load_data(
    self,
    query: str,
    metadata: Dict = {},
    doc_cols: List = [],
    metadata_cols: List = []
)
```

Rather than Optional, I think you can just set defaults as I did above.

Reply:
Please also use the types that I've indicated above, otherwise mypy might complain